### ISSUE TITLE
crash on https://www.hoodamath.com/games/blockswithletterson.html

### ISSUE THREAD

**Issue by hoodamath** [ruffle-rs/ruffle#13125]
### Describe what you were doing

game says major issue before starting

### What does the crash message say?

ruffle has encountered a major issue while trying to display flash content

### Ruffle Version

aug 8

### Affected platform

Self-hosted version

### Operating system

mac os

### Additional information

https://www.hoodamath.com/games/blockswithletterson.html

**Comment by Croworbit** [comment]
`# Error Info
Error name: Error
Error message: panicked at 'No script object for display object', core/src/avm1/activation.rs:404:18
Error stack:
```
339/__wbg_get_imports/imports.wbg.__wbg_new_d258248ed531ff54@moz-extension://eea8366e-8f34-4e21-a7cf-3e2a30f509aa/dist/339.js:2406:21
core::ops::function::FnOnce::call_once{{vtable.shim}}::h3959f346f7109267@:wasm-function[9258]:0x9138dc
std::panicking::rust_panic_with_hook::h7601402c0a383194@:wasm-function[6015]:0x8139ef
core::panicking::panic_fmt::h6dad0405f48e39e2@:wasm-function[9098]:0x90ef87
ruffle_core::display_object::edit_text::EditText::try_bind_text_field_variable::hccd7e0bc619644b9@:wasm-function[3168]:0x673a98
<ruffle_core::display_object::DisplayObject as ruffle_core::display_object::TDisplayObject>::post_instantiation::h97a7898573cd6c18@:wasm-function[3089]:0x662d34
ruffle_core::display_object::movie_clip::MovieClip::instantiate_child::h13825d8aaec8eb73@:wasm-function[1051]:0x3651a5
ruffle_core::display_object::movie_clip::MovieClip::place_object::h4c744372f84dd78f@:wasm-function[1872]:0x511495
<ruffle_core::display_object::movie_clip::MovieClip as ruffle_core::display_object::TDisplayObject>::enter_frame::h2c0f823b3783b793@:wasm-function[800]:0x2a109d
ruffle_core::display_object::movie_clip::MovieClip::instantiate_child::h13825d8aaec8eb73@:wasm-function[1051]:0x3651e1
ruffle_core::display_object::movie_clip::MovieClip::place_object::h4c744372f84dd78f@:wasm-function[1872]:0x511495
<ruffle_core::display_object::movie_clip::MovieClip as ruffle_core::display_object::TDisplayObject>::enter_frame::h2c0f823b3783b793@:wasm-function[800]:0x2a109d
<ruffle_core::display_object::loader_display::LoaderDisplay as ruffle_core::display_object::TDisplayObject>::enter_frame::hbb784dfe2056abd2@:wasm-function[2590]:0x5dda4b
<ruffle_core::display_object::movie_clip::MovieClip as ruffle_core::display_object::TDisplayObject>::enter_frame::h2c0f823b3783b793@:wasm-function[800]:0x2a074f
<ruffle_core::display_object::stage::Stage as ruffle_core::display_object::TDisplayObject>::enter_frame::h38a6c3f6ec7401c1@:wasm-function[3562]:0x6bdae5
ruffle_web::Ruffle::tick::h04b4dcebbcb65025@:wasm-function[760]:0x26b2c6
ruffle_web::Ruffle::new_internal::{{closure}}::{{closure}}::{{closure}}::ha7b686b79dc1ea62@:wasm-function[11210]:0x9556ad
<dyn core::ops::function::FnMut<(A,)>+Output = R as wasm_bindgen::closure::WasmClosure>::describe::invoke::h8b0fe50f371bb3de@:wasm-function[9913]:0x930808
__wbg_adapter_66@moz-extension://eea8366e-8f34-4e21-a7cf-3e2a30f509aa/dist/339.js:294:10
real@moz-extension://eea8366e-8f34-4e21-a7cf-3e2a30f509aa/dist/339.js:271:20`


this is the error I get trying to play

**Comment by n0samu** [comment]
This is an AVM1 game encrypted with MochiCrypt, which uses AVM2. Ruffle does not correctly play AVM1 movies loaded by AVM2 movies right now, but this is being worked on in #11005. If the original AVM1 movie is extracted from the MochiCrypt wrapper and played on its own, it should work fine.

**Comment by Lord-McSweeney** [comment]
Fixed by #11005.
